modules/Saritasa.Build/Saritasa.Build.psm1

```diff
@@ -80,6 +83,11 @@ function Invoke-NugetRestore
     if ($SolutionPath)
     {
         $params += $SolutionPath
+
+        if ($ConfigFile)
+        {
+          $params += @('-ConfigFile', $ConfigFile)
```

Comment:
It should be outside of the `if ($SolutionPath)` to be applied both to solutions and projects.